doc/todo/NEWS.md

```diff
@@ -48,52 +48,63 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
 ### <<HIGHLIGHT2>>
 
 
-## Other New Features
-
-We added even more functionality, which could not make it to the highlights:
+## Bindings
```

Comment:
What was the reason to start with Bindings and not Plugins? But I actually do not know what is more important for people reading the release notes. I assumed it is the plugins but maybe that is wrong.

Reply:
> What was the reason to start with Bindings and not Plugins? 

The main reason was alphabetic order 😊. I changed the arrangement in the latest update.